docs/community/com-query/grafana-connector.md

```diff
@@ -95,8 +102,8 @@ docker run -d  \
     -p 20101:20101 \ # FeatureBase grpc port
     -v "$(pwd)":/var/lib/grafana/plugins \   # plugin directory volume mount
     --name=grafana \
-    -e "GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=molecula-datasource" \   # grafana configuration environment variable
-    grafana/grafana:7.5.9
+    -e "GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=featurebase-datasource" \   # grafana configuration environment variable
+    grafana/grafana:9.4.7
 ```
 
 After any changes to the plugin directory, restart the container with `docker restart grafana`.
```

Comment:
This should be a simple instruction along the lines of:

* Open a terminal and restart the container:
```
docker restart grafana
```